module/Company/src/Company/Model/Job.php

```diff
@@ -233,24 +247,97 @@ public function setDescription($description)
     }
 
     /**
-     * Get the job's company.
-     *
-     * @return Company
+     * Get the job's language.
+     * 
+     * @return string language of the job
+     */
+    public function getLanguage()
+    {
+        return $this->language;
+    }
+
+    /**
+     * Set the job's language.
+     * 
+     * @param string $language language of the job
      */
-    public function getCompany()
+    public function setLanguage($language)
     {
-        return $this->company;
+        $this->language = $language;
     }
 
     /**
-     * Set the job's company.
+     * Get the job's packet.
      *
-     * @param Company company
+     * @return CompanyPacket
      */
-    public function setCompany($company)
+    public function getPacket()
     {
-        $this->company = $company;
+        return $this->packet;
     }
 
+    /**
+     * Set the job's packet.
+     * 
+     * @param CompanyPacket $packet the job's packet
+     */
+    public function setPacket(CompanyPacket $packet)
+    {
+        $this->packet = $packet;
+    }
+
+    // For zend2 forms
+    /**
+     * Returns an array containing all instance variables of this object
+     *
+     */
+    public function getArrayCopy()
+    {
+        $array = get_object_vars($this);
```

Comment:
This might leak unwanted interal object states. It would be better to explicitly create an array using getters.